### PR TITLE
Updated the input plugins reference of hwnd on game start.

### PR DESCRIPTION
**Pull request by skidau**
Fixes the emu wiimotes mouse IR when rendering the game to a separate window.

### CODE REVIEW

File: Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp [hunk 1]
```diff
@@ -43,9 +43,6 @@ ControllerInterface g_controller_interface;
 //
 void ControllerInterface::Initialize(void* const hwnd)
 {
-	if (m_is_init)
-		return;
```
Comment (magumagu) [inline Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp]: Getting rid of just the one use of m_is_init is a little strange... we should either be checking it everywhere, or get rid of it.

Comment (skidau) [inline Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp]: Yes, I was going to do that, but this way, it allows multiple init's but will not shutdown unless it has been initialised.

Comment (magumagu) [inline Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp]: Okay. If multiple init's are okay, you might want to get rid of Reinitialize().

Comment (lioncash) [inline Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp]: Are you sure multiple initializations won't cause memory leaks? If I recall, initializing twice causes this for some input backends. Granted, that may have been fixed as well.

Comment (skidau) [inline Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp]: I have changed it to DeInit if the interface was already initialised to be safe.

**Pull request merged**